src/components/PopupBase/__tests__/__snapshots__/PopupBase.spec.tsx.snap

```diff
@@ -4,7 +4,8 @@ exports[`<PopupBase> that adds basic anchor/popup functionality to rendered comp
 <body>
   <div>
     <button
-      class="Button"
+      class="Button Button--variant_filled Button--context_primary"
```

Comment:
can you check if these buttons are only set in the test, or if we should preserve the neutral look for them?